docs/geedocs/docsrc/BUILD_DOCS.md

```diff
@@ -145,10 +143,12 @@ branch and master.
       Keep the change, which has |release| instead of 5.3.3. This will help
       update the documentation based on the current Open GEE release.
 
-      Merges may become tricky if there are conflicts in the generated HTML.
+      NOTE: Merges may become tricky if there are conflicts in the generated HTML.
       If bringing 5.3.2 changes into master there will be 5.3.2
-      HTML changes. You can keep changes in the 5.3.2 branch while resolving
-      conflicts. Resolve conflicts in the RST and generated HTML for master
+      HTML changes. You can keep 5.3.2 html changes from the 5.3.2 branch while resolving
+      conflicts. If there are conflicts in 5.3.3 html files, regenerating the html files
+      is the best way to solve it.
+      If there are conflicts in the RST, resolve them and generate HTML for master
```

Comment:
take away the 'for master'  These instructions can be followed for any merge into release branches too.  I think we wanted these instructions to be generic for any branch.  Basically only worry about resolving conflicts in non-generated code and then re-generate the generated code to then resolve conflicts in that code.